common/persistence/serializer_test.go

```diff
@@ -58,8 +58,8 @@ func (s *cadenceSerializerSuite) SetupTest() {
 	s.Assertions = require.New(s.T())
 }
 
+// TODO: update tests to cover asyncworkflowconfig
```

Comment:
Do you mind adding the tests?

Reply:
[this test body](https://github.com/uber/cadence/blob/master/common/persistence/serializer_test.go#L210-L583) is not ideal. I will convert this to proper table test and simplify the body in a follow up PR